dist/less/catalogParameters.less

```diff
@@ -8,6 +8,19 @@
     padding-left: 0;
   }
   &.readonly {
+    .control-label {
+      padding-right: 0;
+      word-break: break-word;
```

Comment:
I don't think this is a standard `word-break` value. Do you mean `word-wrap: break-word;`?

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
Yes, sorry.